### DISCUSSION TITLE
EJBCA-CE Database Tables and Indexes

### DISCUSSION THREAD

**Discussion by Cadjoe** [Keyfactor/ejbca-ce#550] [Q&A] [answered]
Hi Folks!

I on the step [Create the database](https://doc.primekey.com/ejbca/ejbca-installation/creating-the-database) using PostgreSQL. My setup requires a remote connection to the database. I have the database and user created on the PostgreSQL host and remote authentication for the user and access provisioned and tested successfully. 

Now, on step 5, under the PostgreSQL section—Create EJBCA tables and indexes, the doc assumes the `.sql files, DB, and EJBCA` installation are all on the same server. See below:
```
$ psql -U ejbca_user -W ejbcadb
> psql -hlocalhost -U ejbca_user -d ejbcadb < /opt/ejbca/doc/sql-scripts/create-tables-ejbca-postgres.sql
> psql -hlocalhost -U ejbca_user -d ejbcadb < /opt/ejbca/doc/sql-scripts/create-index-ejbca.sql
> \q
```
I am hoping I could download the `.sql` table and index files and run them separately against the DB. However, I can't seem to find a download location anywhere for those files.

Thanks in advance for your time and assistance.

**Comment by primetomas** [comment, marked as answer]
I'm not sure why that is there in the documentation for postgres specifically. For the other databases it doesn't say you have to create tables manually. I'll remove those.
Tables are created automatically when EJBCA is deployed the first time (requires create table privileges obviously).
There is a separate section for creating indexes anyhow, so just ignore those two SQL files in the description.

To see them though, they are right here in the code: https://github.com/Keyfactor/ejbca-ce/tree/main/doc/sql-scripts

**Reply by Cadjoe** [reply]
Got it and thanks for the prompt response @primetomas.
